pom.xml

```diff
@@ -22,11 +22,13 @@
     <parent>
         <groupId>io.confluent</groupId>
         <artifactId>kafka-connect-storage-common-parent</artifactId>
-        <version>6.0.0-SNAPSHOT</version>
+        <version>10.0.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>kafka-connect-hdfs</artifactId>
+    <version>10.0.0-SNAPSHOT</version>
     <packaging>jar</packaging>
+    <version>10.0.0-SNAPSHOT</version>
```

Comment:
duplicate

Reply:
Nice catch. I cherry-picked a commit and this made it in.